src/ReactionMechanismSimulator.jl

```diff
@@ -30,6 +31,7 @@ module ReactionMechanismSimulator
         copy!(falloff,pyimport_conda("rmgpy.kinetics.falloff","rmg","rmg"))
         copy!(chebyshev,pyimport_conda("rmgpy.kinetics.chebyshev","rmg","rmg"))
         copy!(solvation,pyimport_conda("rmgpy.data.solvation","rmg","rmg"))
+        copy!(quantity,pyimport_conda("rmgpy.quantity","rmg","rmg"))
```

Comment:
Can we use Unitful.jl https://github.com/PainterQubits/Unitful.jl (already a dependency) instead? Long term I'd like to phase RMG out as a dependency.

Reply:
I wasn't aware of this package, thanks for sharing, I will use this instead